driver/driver.feature

```diff
@@ -0,0 +1,1265 @@
+
+  ###########
+  # QUERIES #
+  ###########
+
+  Scenario: Driver processes ok query answers correctly
+    Given connection open schema transaction for database: typedb
+    When get answers of typeql schema query
+      """
+      define entity person;
+      """
+    Then answer type is: ok
+    Then answer type is not: concept rows
+    Then answer type is not: concept trees
+    Then result is a successful ok
+    Then transaction commits
+
+
+    # TODO: Test optionals when introduced
+  Scenario: Driver processes concept row query answers correctly
+    Given connection open schema transaction for database: typedb
+    Given typeql schema query
+      """
+      define entity person owns name @key; attribute name, value string;
+      """
+    When get answers of typeql read query
+      """
+      match entity $p;
+      """
+    Then answer type is: concept rows
+    Then answer type is not: ok
+    Then answer type is not: concept trees
+    Then answer query type is read
+    Then answer query type is not schema
+    Then answer query type is not write
+    Then answer size: 1
+    Then answer column names are:
+      | p |
+    Then answer get row(0) get entity type(p) get label: person
+    Then answer get row(0) get entity type by index of variable(p) get label: person
+
+    When get answers of typeql read query
+      """
+      match entity $p; attribute $n;
+      """
+    Then answer type is: concept rows
+    Then answer type is not: ok
+    Then answer type is not: concept trees
+    Then answer query type is read
+    Then answer query type is not schema
+    Then answer query type is not write
+    Then answer size: 1
+    Then answer column names are:
+      | p |
+      | n |
+    Then answer get row(0) get entity type(p) get label: person
+    Then answer get row(0) get attribute type(n) get label: name
+    Then answer get row(0) get entity type by index of variable(p) get label: person
+    Then answer get row(0) get attribute type by index of variable(n) get label: name
+
+    When typeql schema query
+      """
+      define attribute age, value long;
+      """
+    When get answers of typeql read query
+      """
+      match entity $p; attribute $n;
+      """
+    Then answer type is: concept rows
+    Then answer type is not: ok
+    Then answer type is not: concept trees
+    Then answer query type is read
+    Then answer query type is not schema
+    Then answer query type is not write
+    Then answer size: 2
+    Then answer column names are:
+      | p |
+      | n |
+    Then answer get row(0) get entity type(p) get label: person
+    Then answer get row(0) get attribute type(n) get label: name
+    Then answer get row(0) get entity type by index of variable(p) get label: person
+    Then answer get row(0) get attribute type by index of variable(n) get label: name
+    Then answer get row(1) get entity type(p) get label: person
```

Comment:
I actually rely on ordering here a little bit as I get a specific row. Not sure if I can even if it's that simple (just two consecutive attribute types), can additionally sort it on the implementation side.

Reply:
Just add a sort to the query right?

Reply:
If we have it...

Reply:
Good point. It works for now, we can add sorting if it starts failing, I guess, because I'm not sure about the syntax atm. It's only needed in this single test.